### PR TITLE
fix: themeing bugs in OpenID4VCP Proof Requests

### PR DESCRIPTION
**Pull request by MosCD3**
# Summary of Changes

1. Fix themeing issues in Proof request screen
2. Some cleanup and refactor
No breaking changes

# Screenshots, videos, or gifs

Replace this text with embedded media for UI changes if they are included in this PR. If there are none, simply enter N/A

# Breaking change guide

Replace this text with any breaking changes included in this PR along with how to address them in downstream projects. If there are none, simply enter N/A

# Related Issues

Replace this text with issue #'s that are relevant to this PR. If there are none, simply enter N/A

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [ ] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this)
- [ ] If applicable, screenshots, gifs, or video are included for UI changes
- [ ] If applicable, breaking changes are described above along with how to address them
- [ ] Updated documentation as needed for changed code and new or modified features
- [ ] Added sufficient [tests](../packages/legacy/core/__tests__/) so that overall code coverage is not reduced

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated checks have passed

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2887,6 +2887,7 @@ __metadata:
     react-native-svg-transformer: ^0.14.3
     react-native-tcp-socket: ^6.0.6
     react-native-toast-message: ^2.1.6
+    react-native-url-polyfill: ^2.0.0
```
Comment (cvarjao) [inline yarn.lock]: Update lock file to reflect unused depnedency

File: packages/legacy/core/App/modules/openid/components/CredentialRowCard.tsx [hunk 2]
```diff
@@ -26,23 +31,28 @@ export function OpenIDCredentialRowCard({ name, issuer, bgColor, onPress }: Cred
     },
     issuerBadge: {
       borderRadius: 8,
-      width: '30%',
+      width: badgeHeight,
```
Comment (al-rosenthal) [inline packages/legacy/core/App/modules/openid/components/CredentialRowCard.tsx:34]: was using badgeHeight twice intentional?

File: packages/legacy/core/App/modules/openid/screens/OpenIDProofPresentation.tsx [hunk 3]
```diff
@@ -222,7 +243,7 @@ const OpenIDProofPresentation: React.FC<OpenIDProofPresentationProps> = ({
       </ScrollView>
       {footer()}
 
-      <ProofRequestAccept visible={acceptModalVisible} proofId={''} confirmationOnly = {true} />
+      <ProofRequestAccept visible={acceptModalVisible} proofId={''} confirmationOnly={true} />
```
Comment (al-rosenthal) [inline packages/legacy/core/App/modules/openid/screens/OpenIDProofPresentation.tsx:246]: no proof id?

**Review by al-rosenthal — COMMENTED** [review]
2 small comments and needs the latest changes, looks good otherwise

File: packages/legacy/core/App/localization/en/index.ts [hunk 4]
```diff
@@ -676,6 +677,7 @@ const translation = {
     "NoInfoShared": "No information was shared",
     "YourInfo": "Your information was not shared",
     "YouCantRespond": "You can't respond due to the following reasons. Please address them before continuing.",
+    "CredentailNotInWallet": "This credential is not present in your wallet.",
```
Comment (bryce-mcmath) [inline packages/legacy/core/App/localization/en/index.ts]: Should be `CredentialNotInWallet` I think

File: packages/legacy/core/App/modules/openid/utils/utils.tsx [hunk 5]
```diff
@@ -17,11 +18,11 @@ export function sanitizeString(str: string) {
...
 export function getHostNameFromUrl(url: string) {
-  //TODO: Find more elegant way to extract host name
-  // const urlRegex = /^(.*:)\/\/([A-Za-z0-9-.]+)(:[0-9]+)?(.*)$/
-  // const parts = urlRegex.exec(url)
-  // return parts ? parts[2] : undefined
-  return url.split("https://")[1]
+  try {
+    return getDomainFromUrl(url)
+  } catch (error) {
+    return 'undefined!'
```
Comment (bryce-mcmath) [inline packages/legacy/core/App/modules/openid/utils/utils.tsx]: This string is kind of a strange return for an error. Wouldn't `null` or an actual `undefined` be better?

**Review by bryce-mcmath — CHANGES_REQUESTED** [review]

File: packages/legacy/core/App/modules/openid/components/CredentialRowCard.tsx (hunk 2 above)
Comment (MosCD3) [reply packages/legacy/core/App/modules/openid/components/CredentialRowCard.tsx:34]: yes, its supposed to be squared

File: packages/legacy/core/App/modules/openid/screens/OpenIDProofPresentation.tsx (hunk 3 above)
Comment (MosCD3) [reply packages/legacy/core/App/modules/openid/screens/OpenIDProofPresentation.tsx:246]: No, its not there in OpenID

File: packages/legacy/core/App/localization/en/index.ts (hunk 4 above)
Comment (MosCD3) [reply packages/legacy/core/App/localization/en/index.ts]: done

File: packages/legacy/core/App/modules/openid/utils/utils.tsx (hunk 5 above)
Comment (MosCD3) [reply packages/legacy/core/App/modules/openid/utils/utils.tsx]: done

**Review by bryce-mcmath — APPROVED** [review]

**Pull request merged**